src/main/java/mate/academy/springboot/datajpa/dto/request/CategoryRequestDto.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.springboot.datajpa.dto.request;
+
+import javax.validation.constraints.NotNull;
+import lombok.Data;
+
+@Data
```

Comment:
I'd recommend using specific Lombok annotations here instead of general Data. You can fix the same issue in all places